projects/slap_manipulation/readme.md

```diff
@@ -0,0 +1,13 @@
+# Spatial-Language Attention Policies for Efficient Robot Learning
+
+## Task list
```

Comment:
Good idea, but wouldn't we need to go through any "data release" clearance. I am imagining this would be rather small data to experiment with, but good to check.